docs/casestudies/adopters.md

```diff
@@ -11,16 +11,16 @@ To join this list, please follow [these instructions](https://github.com/karmada
-| Zhejiang Lab              | https://www.zhejianglab.com/  | Use Karmada to build multi-cluster platform                                                                                                                                                                             | TBD                                                                           |
\ No newline at end of file
+| Organization/Company Name | Website                       | Usage Scenario                                                                                                                                                                                                           | CaseStudy                                                                     |
+|---------------------------|-------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------|
+| Cibfintech.com            | https://www.cibfintech.com/   | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada                                                                                                                                             | TBD                                                                           |
+| DaoCloud.io               | https://www.daocloud.io/      | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada                                                                                                                                             | TBD                                                                           |
+| Ecloud                    | https://ecloud.10086.cn/      | Use Karmada to build a steady and elastic multi-cluster platform for managing workloads                                                                                                                                  | TBD                                                                           |
+| Huawei                    | https://www.huaweicloud.com/  | Built the [Ubiquitous Cloud Native Service](https://support.huaweicloud.com/intl/en-us/productdesc-ucs/ucs_productdesc_0001.html) that run applications across regions and clouds with intelligent traffic distribution  | TBD                                                                           |
+| Hurricane Engines         | https://orang.cloud/          | The [ORANG](https://orang.cloud/) `low-code` platform provides the service for users to build powerful apps quickly with zero lines of code, [ORANG](https://orang.cloud/) use Karmada to deploy apps across multi-cloud | [Karmada in AIML INSTITUTE](ci123.md)                                         |
```

Comment:
A small typo.
[ORANG](https://orang.cloud/) uses Karmada to deploy apps across multi-cloud.

Reply:
done.